mmdet3d/ops/sparse_block.py

```diff
@@ -43,24 +48,44 @@ def __init__(self,
             norm_cfg=norm_cfg)
 
     def forward(self, x):
-        identity = x.features
+        if spconv2_is_avalible:
+            identity = x.features
+
+            out = self.conv1(x)
+            out = out.replace_feature(self.bn1(out.features))
+            out = out.replace_feature(self.relu(out.features))
+
+            out = self.conv2(out)
+            out = out.replace_feature(self.bn2(out.features))
+            out = out.replace_feature(self.relu(out.features))
+
+            out = self.conv3(out)
+            out = out.replace_feature(self.bn3(out.features))
 
-        out = self.conv1(x)
-        out.features = self.bn1(out.features)
-        out.features = self.relu(out.features)
+            if self.downsample is not None:
+                identity = self.downsample(x)
 
-        out = self.conv2(out)
-        out.features = self.bn2(out.features)
-        out.features = self.relu(out.features)
+            out = out.replace_feature(out.features + identity)
+            out = out.replace_feature(self.relu(out.features))
```

Comment:
These two lines are the only differences between if-else? May simplify the codes